src/utils.ts

```diff
@@ -50,7 +53,7 @@ export function getFileData(filePath: string, fileName: string): Promise<string>
  * @param field The name of the field to return
  * @returns {string} The value of the header field
  */
-export function getHeaderValue(req: express.Request, fieldName: string): string {
+export function getHeaderValue(req: Request, fieldName: string): string | undefined {
```

Comment:
What do you think about this solution? I think `hasOwn` was anyway too much, as we can't asume that from `request`.